.rubocop.yml

```diff
@@ -6,7 +6,7 @@ inherit_mode:
     - AllowedNames
 
 AllCops:
-  TargetRubyVersion: 2.3
+  TargetRubyVersion: 2.4
```

Comment:
Not sure if you intended to remove 2.3 compatibility based on the commit message?

Reply:
My bad.

This was a consequence of `tpm-key_attestation` not supporting ruby 2.3.

Reply:
I considered extra effort to support ruby 2.3 in `tpm-key_attestation` but later realized it could be reasonable to drop ruby 2.3 support given it's going to be a year since 2.3 end of life this March.